### ISSUE TITLE
Feature Request - Data on player screen.

### ISSUE THREAD

**Issue by NripPrajapat** [anilbeesetti/nextplayer#1131]
Please provide an option to show 'time', remaining time/elapsed time and 'battery percentage' on player screen, in corners.

**Comment by kubalav** [comment]
I agree with this request and would also like to ask for the remaining time to be recalculated according to the set playback speed. This is a feature that prevents me from switching to Next Player.
